KPCTabsControlDemo/PaneViewController.swift

```diff
@@ -129,5 +135,10 @@ class PaneViewController: NSViewController, TabsControlDataSource, TabsControlDe
         let range = index..<index+1
         self.items.replaceRange(range, with: [newItem])
     }
+
+    func tabsControl(control: TabsControl, canSelectItem item: AnyObject) -> Bool {
+
+        return (item as! Item) != self.items.first
+    }
 }
```

Comment:
Sounds good. But I am wondering now: shouldn't we also include a way in `Style` (or `ThemeStyle`...) distinguish tabs that are not selectable? With something like a slightly light title text color...